lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -53,21 +61,33 @@ public class BulkMigrateVcsHandler implements RequestHandler<Request, BatchRepor
     private final VerifiableCredentialService verifiableCredentialService;
     private final EvcsClient evcsClient;
     private final ForkJoinPoolFactory forkJoinPoolFactory;
+    private final ConfigService configService;
+    private final AuditService auditService;
 
     public BulkMigrateVcsHandler(
             ScanDynamoDataStore<ReportUserIdentityItem> reportUserIdentityScanDynamoDataStore,
             VerifiableCredentialService verifiableCredentialService,
             EvcsClient evcsClient,
-            ForkJoinPoolFactory forkJoinPoolFactory) {
+            ForkJoinPoolFactory forkJoinPoolFactory,
+            ConfigService configService,
+            AuditService auditService) {
         this.reportUserIdentityScanDynamoDataStore = reportUserIdentityScanDynamoDataStore;
         this.verifiableCredentialService = verifiableCredentialService;
         this.evcsClient = evcsClient;
         this.forkJoinPoolFactory = forkJoinPoolFactory;
+        this.configService = configService;
+        this.auditService = auditService;
     }
 
     @ExcludeFromGeneratedCoverageReport
     public BulkMigrateVcsHandler() {
-        var configService = ConfigService.create();
+        this(ConfigService.create());
+    }
```

Comment:
Do we need this extra constuctor? What's the use case of having a constructor that allows the ConfigService to be injected on its own?

Reply:
Just to inject config service, We did similar in other handler example `CheckExistingIdentityHandler`

Reply:
I don't think this new constructor is used though - and we have the ability to inject a config service in the all args constructor, which is the one used by the tests.

Looking at that constructor in check existing identity - it's also unused. We should probably remove it...